modules/imgproc/include/opencv2/imgproc/hal/hal.hpp

```diff
@@ -101,7 +101,7 @@ CV_EXPORTS void morph(int op, int src_type, int dst_type,
 CV_EXPORTS void resize(int src_type,
                        const uchar * src_data, size_t src_step, int src_width, int src_height,
                        uchar * dst_data, size_t dst_step, int dst_width, int dst_height,
-                       double inv_scale_x, double inv_scale_y, int interpolation);
+                       double inv_scale_x, double inv_scale_y, int interpolation, int coordinate);
```

Comment:
We should not break HAL API/ABI.
Don't modify exited function, create a new function instead.